src/vos/evtree.c

```diff
@@ -1823,6 +1843,70 @@ evt_select_node(struct evt_context *tcx, const struct evt_rect *rect,
 	return rc < 0 ? nd1 : nd2;
 }
 
+/** Expand the dynamic tree root node if it is currently full and not
+ *  already at full size
+ */
+static inline int
+evt_node_extend(struct evt_context *tcx, struct evt_trace *trace)
+{
+	struct evt_node *nd_cur;
+	int              rc;
+	uint8_t          old_order;
+	void            *new_node;
+	int              old_size;
+	umem_off_t       new_off;
+
+	if (tcx->tc_order == tcx->tc_max_order)
```

Comment:
If EVT_FEAT_DYNAMIC_ROOT is not supported, then tc_max_order is zero, we need to return under such case?

Reply:
tcx->tc_max_order gets set to root->tr_order in that case so this should always return.